### PR TITLE
Create Database Connection

### PR DESCRIPTION
**Pull request by Abiodun-Shittu**
## Create Database Connection
- Using MongoDB

### CODE REVIEW

File: database/db.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const connection = async() => {
+	try {
+		await mongoose.connect(process.env.MONGO_URL)
```
Comment (xyluz) [inline database/db.js]: Please add .env.example

**Review by xyluz — CHANGES_REQUESTED** [review]

File: database/db.js (hunk 1 above)
Comment (Abiodun-Shittu) [reply database/db.js]: okay, will do that now

Comment (Abiodun-Shittu) [reply database/db.js]: Resolved

**Comment by xyluz** [comment]
Please resolve conflicts

**Comment by Abiodun-Shittu** [comment]
> Please resolve conflicts

conflict resolved

**Pull request merged**